### PR TITLE
Fixed first turn and more

### PR DESCRIPTION
**Pull request by NomadCHK**
Tested by the night and runs ok. even the double quotes on the time_limit string is ok. 

please @AdriaGual test this version and lets improve it. Please add back your beautiful logs icons.

Thank youu (=

### CODE REVIEW

**Comment by AdriaGual** [comment]
@NomadCHK Gonna test this for some hours, but the code looks good!

File: bot.py [hunk 1]
```diff
@@ -198,18 +196,35 @@ def run_script(self):
                 screenshot, self.template_images["CROSS_BUTTON"], "Cross button"
             )
 
-    def play_turn(self):
+    def play_turn(self, is_first_turn=False):
         if not self.running:
             return False
         self.log_callback("Start playing my turn...")
+
         self.add_energy_to_pokemon()
```
Comment (AdriaGual) [inline bot.py:204]:
```suggestion
    if not is_first_turn:
        self.add_energy_to_pokemon()
```

Comment (AdriaGual) [reply bot.py:204]: If its the first turn you shouldn't attach an energy

**Review by AdriaGual — CHANGES_REQUESTED** [review]

**Comment by NomadCHK** [comment]
Still needs to improve end battle (need to improve wait times, time to click next, time to "thank you" button etc).
Also @AdriaGual lets define a default formatting? what linter and formating you use? lets use black or ruff. We can create the standards  on .vscode folder

**Comment by NomadCHK** [comment]
this PR may will solve #7  and #9

**Comment by AdriaGual** [comment]
@NomadCHK Yeah, we can go with black since it's the standard, will merge this for now so we can close the issues and then we can focus on other improvements.

The timings is also one thing that it is taking through my nerves, but slowly we will have a version that doesn't take too long that the turn ends and at the same time it checks properly everything haha

Huge thanks with your contributions as always ❤️ 🥇

**Review by AdriaGual — APPROVED** [review]
LGTM

**Pull request merged**